SingularityService/src/test/java/com/hubspot/singularity/data/StateManagerTest.java

```diff
@@ -74,4 +77,48 @@ public void itDoesntFlagPendingRequestsForUnderOrOverProvisioning() {
     Assert.assertEquals(0, state.getOverProvisionedRequests());
     Assert.assertEquals(0, state.getUnderProvisionedRequests());
   }
+
+  @Test
+  public void itDoesntCountInstancesOverLimitInOnDemandLateTasks() {
+    initOnDemandRequest();
+    SingularityRequest request = requestResource.getRequest(requestId, singularityUser).getRequest();
+    requestResource.postRequest(request.toBuilder().setInstances(Optional.of(2)).build(), singularityUser);
+    initFirstDeploy();
+
+    configuration.setDeltaAfterWhichTasksAreLateMillis(TimeUnit.MILLISECONDS.toMillis(10));
```

Comment:
shouldn't need MILLISECONDS.toMillis, can just use the number. Also, why not set to `0`? Outcome otherwise may be different depending on how fast the machine running the test is